Dizzy/Coordinators/LoginCoordinator.swift

```diff
@@ -99,17 +99,17 @@ extension LoginCoordinator: LoginVMNavigationDelegate, SignInWithDizzyVMNavigati
     }
     
     func navigateToAppInfoScreen(type: AppInfoType) {
-        var baseUrl = "https://dizzy.co.il/"
+        var baseUrl = "https://"
```

Comment:
leave this as it was and all the other lines except line 108 write `beUrl = "https://wa.me/972542943889"